fv3core/stencils/ytp_v.py

```diff
@@ -1,230 +1,165 @@
-from gt4py.gtscript import PARALLEL, computation, interval
+from gt4py import gtscript
+from gt4py.gtscript import (
+    __INLINED,
+    PARALLEL,
+    computation,
+    horizontal,
+    interval,
+    region,
+)
 
 import fv3core._config as spec
-import fv3core.stencils.yppm as yppm
-import fv3core.utils.gt4py_utils as utils
 from fv3core.decorators import gtstencil
-
-from .yppm import (
-    compute_al,
-    final_flux,
-    fx1_fn,
-    get_b0,
-    get_bl,
-    get_br,
-    is_smt5_mord5,
-    is_smt5_most_mords,
-)
+from fv3core.stencils import yppm
+from fv3core.utils.typing import FloatField
 
 
-sd = utils.sd
+@gtscript.function
+def get_flux_v_ord8plus(
+    q: FloatField, c: FloatField, rdy: FloatField, bl: FloatField, br: FloatField
+):
+    b0 = bl + br
+    cfl = c * rdy[0, -1, 0] if c > 0 else c * rdy
+    fx1 = yppm.fx1_fn(cfl, br, b0, bl)
+    return q[0, -1, 0] + fx1 if c > 0.0 else q + fx1
 
 
-@gtstencil()
-def get_flux_v_stencil(
-    q: sd, c: sd, al: sd, rdy: sd, bl: sd, br: sd, flux: sd, mord: int
+def get_flux_v(
+    q: FloatField, c: FloatField, rdy: FloatField, bl: FloatField, br: FloatField
 ):
-    with computation(PARALLEL), interval(...):
-        b0 = get_b0(bl=bl, br=br)
-        smt5 = is_smt5_mord5(bl, br) if mord == 5 else is_smt5_most_mords(bl, br, b0)
-        tmp = smt5[0, -1, 0] + smt5 * (smt5[0, -1, 0] == 0)
-        cfl = c * rdy[0, -1, 0] if c > 0 else c * rdy
-        fx0 = fx1_fn(cfl, br, b0, bl)
-        # TODO: add [0, 0, 0] when gt4py bug is fixed
-        flux = final_flux(c, q, fx0, tmp)  # noqa
+    from __externals__ import mord
```

Comment:
For readability and documentation, I would suggest doing this and then using the longer name below. We can consider refactoring the namelist name itself much later, for now it's worth staying consistent with Fortran.
```suggestion
    from __externals__ import mord
    momentum_advection_scheme = mord
```

Reply:
Let's do this in a follow-up PR.